config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```

Comment:
1. Config this file into .gitignore then run bellow command line --> do not push this file to pull request
`git rm --cached config/database.yml `
2. Create a new file with name "config/database.yml.example" --> this file using push to pull request

Reply:
How to do this?

Reply:
this file will not be up to GIt, please

```git rm --cached config/database.yml```
and in .gitignore add
```config/database.yml```

Reply:
![Screenshot from 2021-08-19 15-38-37](https://user-images.githubusercontent.com/87749730/130027961-0a2e8e03-28e4-47ce-a9db-130836a3b15b.png)
why does it appear like this?